include/picongpu/param/transitionRadiation.param

```diff
@@ -0,0 +1,175 @@
+/* Copyright 2013-2019 Rene Widera, Richard Pausch, Finn-Ole Carstens
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/** @file
+ *
+ * Definition of frequency space, number of observers, filters, form factors and
+ * window functions of the radiation plugin.
+ *
+ * All values set here determine what the radiation plugin will compute.
+ * The observation direction is defined in a seperate file `radiationObserver.param`.
+ * On the comand line the plugin still needs to be called for each species the radiation
+ * should be computed for.
+ */
+#pragma once
+
+/**
+ * radiation verbose level:
+ * 0=nothing, 1=physics, 2=simulation_state, 4=memory, 8=critical
+ */
+#define PIC_VERBOSE_RADIATION 3
```

Comment:
Do you use the same Verbose level scheme? Than please be aware that with #3004 it went to a sub namespace.